pandapower/file_io.py

```diff
@@ -302,6 +308,10 @@ def from_json_string(json_string, convert=False, encryption_key=None, elements_t
         >>> net = pp.from_json_string(json_str)
 
     """
+    if not replace_elements is None:
```

Comment:
# better would be:
if replace_elements is not None:

Reply:
thanks. Adapted it.

Reply:
added a changelog